modules/ROOT/pages/amazon/amazon-dynamodb-connector-studio.adoc

```diff
@@ -0,0 +1,153 @@
+= Use Studio to Configure Amazon DynamoDB Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<conigure-other-fields,Configure the other connector fields>>.
+
+If you are new to configuring connectors in Studio, see xref:introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:amazon/amazon-dynamodb-connector-reference.adoc[Amazon DynamoDB Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Anypoint Studio, from the main menu, select *File > New > Mule Project*.
```

Comment:
I know I've seen this a million times, but just now I wonder if we need "from the main menu"?